packages/suite/src/actions/suite/__fixtures__/suiteActions.ts

```diff
@@ -305,6 +305,36 @@ const selectDevice = [
     },
 ];
 
+const selectNewlyConnectedDevice = [
+    {
+        description: `select a new device`,
+        state: {
+            device: { devices: [] },
+        },
+        newlyConnectedDevice: { ...CONNECT_DEVICE },
```

Comment:
Tiny nit: spread not needed.
```suggestion
        newlyConnectedDevice: CONNECT_DEVICE,
```

Reply:
[fixup! chore(suite-common): select device in device connect thunk](https://github.com/trezor/trezor-suite/pull/21383/commits/0e7e4efd3ba6816dd7ad460eb2f19ce2480d3331)